servant/src/Servant/Types/Internal/ResponseList.hs

```diff
@@ -0,0 +1,17 @@
+{-# LANGUAGE DeriveTraversable #-}
+
+-- | This module offers other servant libraries a minimalistic HTTP response type.
+--
+-- It is purely an internal API and SHOULD NOT be used by end-users of Servant.
+module Servant.Types.Internal.ResponseList where
```

Comment:
why is it called `ResponseList`?

Reply:
Ah, it used to probably host stuff in relation with IsResponseList but does not anymore. I'll rename it to `Response`.